src/test/ui/panic-runtime/auxiliary/exit-success-if-unwind-msvc-no-std.rs

```diff
@@ -0,0 +1,32 @@
+// compile-flags:-C panic=unwind
```

Comment:
It would be good to add a comment explaining what this helper crate does at a high level.